talent/rfcs/0001-healthcheck.md

```diff
@@ -0,0 +1,36 @@
+---
+created-on: 2021-06-11
+freshness: 2021-06-11
+owner: @gilbrookiegirl
+---
+
+[La version française suit.](#bilan-de-santé)
+
+# Health Check
+
+## Summary
+The benefits of employee engagement are well [documented](https://www.kevinkruse.com/employee-engagement-research-master-list-of-29-studies/) The Canadian public service conducts  the [Public Service Employee Survey (PSES) annually](https://www.canada.ca/en/treasury-board-secretariat/services/innovation/public-service-employee-survey/2019-public-service-employee-survey-pses/highlights-2019-pses.html). This survey measures federal government employees’ opinions about their engagement, leadership, workforce, workplace, workplace well-being and compensation. The survey provides a lot of important [data](https://www.canada.ca/en/treasury-board-secretariat/services/innovation/public-service-employee-survey/2019-public-service-employee-survey-pses/highlights-2019-pses.html). However, the PSES results are released once a year, typically three to four months following the close of the survey.
```

Comment:
we have to be careful for not sending referrals to business organizations. Could we find a non-profit, research paper or other government article about employee engagement impact?